### ISSUE TITLE
"Unknown Type: TMyClass"

### ISSUE THREAD

**Issue by arvanus** [gabrielbaltazar/gbswagger#26]
Bom dia pessoal, estou tendo um problema numa rota que retorna um um objeto com outros objetos dentro, o swagger não reconhece um dos retornos, segue um exemplo de retorno esperado, e o JSON da doc. do swagger:

```
{
  "resposta": [
    "GOL 2017 1.6"
  ],
  "completa": {
    "placa": "AAA0000",
    "cor": "PRATA",
    "marca": "VOLKSWAGEN",
    "modelos": [
      {
        "modelo": "GOL",
        "cilindradas": [
          "1.6"
        ]
      }
    ],
    "anoFabricacao": 2017,
    "anoModelo": 2017
  }
}

```

[swagger_json.txt](https://github.com/gabrielbaltazar/gbswagger/files/10923013/swagger_json.txt)
Nota: Esse campo `modelo` é um `TArray<TModelosVeiculo>`

**Comment by gabrielbaltazar** [comment]
Bom dia, o GBSwagger não reconhece o TArray de objetos ainda. nesse caso você precisaria utilizar como TObjectList<TModelosVeiculo>

**Comment by arvanus** [comment]
Certo, se puder me dar um resuminho do que precisa depois eu olho pra fazer um patch

**Comment by arvanus** [comment]
Patch no PR #39
